From 34b8d92c5139b37322548cc41c5c3a788c51d3ad Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Tue, 2 May 2017 12:55:23 +0300
Subject: [PATCH] tools: fix build fail

Build fail fix for CONFIG_ENV_IS_IN_SPI_FLASH

Signed-off-by: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
---
 tools/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tools/Makefile b/tools/Makefile
index 4bbb153..12719ac 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -211,7 +211,8 @@ HOST_EXTRACFLAGS += -include $(srctree)/include/libfdt_env.h \
 		-DCONFIG_SYS_TEXT_BASE=$(CONFIG_SYS_TEXT_BASE) \
 		-DUSE_HOSTCC \
 		-D__KERNEL_STRICT_NAMES \
-		-D_GNU_SOURCE
+		-D_GNU_SOURCE \
+		-include $(srctree)/include/generated/autoconf.h
 
 __build:	$(LOGO-y)
 
-- 
1.9.1

